examples/run_examples.py

```diff
@@ -22,6 +22,7 @@
 
 from daal4py import __daal_link_version__ as dv, __has_dist__
 daal_version = tuple(map(int, (dv[0:4], dv[4:8])))
+print('DAAL version:', daal_version)
```

Comment:
remember delete

Reply:
I think this is legit print to show DAAL version used for examples run and i think it worth having it in longterm